components/push/storage/src/schema.sql

```diff
@@ -8,6 +9,16 @@ CREATE TABLE IF NOT EXISTS push_record (
     ctime              INTEGER  NOT NULL,
     app_server_key     TEXT,
     native_id          TEXT,
-    PRIMARY KEY (uaid, channel_id)
+    PRIMARY KEY
+(uaid, channel_id)
 );
+
+DROP INDEX IF EXISTS channel_id_idx;
 CREATE UNIQUE INDEX channel_id_idx ON push_record(channel_id);
```

Comment:
Unless I'm missing something, `UNIQUE` implies the existence of a `UNIQUE INDEX`, so I don't think we need this.